moon/powerups/server/regen.moon

```diff
@@ -1,49 +1,52 @@
-MAX_HP = 150
-POWERUP_DURATION = 300 -- In seconds
-REGEN_INTERVAL = 0.1 -- How often to apply the regen, in seconds
-REGEN_AMOUNT = 3 -- How much health to apply every REGEN_INTERVAL
-REGEN_SOUND = "items/medcharge4.wav"
+{get: getConf} = CFCPowerups.Config
 
 export RegenPowerup
 class RegenPowerup extends BasePowerup
     @powerupID: "powerup_regen"
 
-    @powerupWeights: {
+    @powerupWeights:
         tier1: 1
         tier2: 1
         tier3: 1
         tier4: 1
-    }
 
     new: (ply) =>
         super ply
 
         @timerName = "CFC_Powerups-Regen-#{ply\SteamID64!}"
 
-        timerDuration = POWERUP_DURATION / REGEN_INTERVAL
-        timer.Create @timerName, REGEN_INTERVAL, timerDuration, @PowerupTick!
+        duration = getConf "regen_duration"
+        interval = getConf "regen_interval"
 
-        @RegenSound = CreateSound(@owner, REGEN_SOUND)
+        timerDuration = duration / interval
+        timer.Create @timerName, interval, timerDuration, @PowerupTick!
```

Comment:
Pretty sure you don't call the function when creating a timer, also can't use the @ reference in a timer I think
```suggestion
        timer.Create @timerName, interval, timerDuration, -> @PowerupTick!
```